src/frontend/src/components/ActionsMenu.tsx

```diff
@@ -13,9 +13,12 @@ export type ButtonInfo = {
 
 interface ActionsMenuProps {
   buttons: ButtonInfo[];
+  title?: string;
+  marginTop?: string;
+  style?: any;
```

Comment:
I realized there was no way we could assign a CSS property (SxProps<Theme>) in ActionsMenu component. 

I tried to give this `style` prop a `SxProps<Theme>` as a type, but kept getting errors, so I gave it as `any` type for now. Lmk if there's better to do this job.

Reply:
you don't need to give it a styles prop (in fact you definitely shouldn't since we want this component styled a particular way), so you should remove that. You can just wrap it in a div wherever you need to apply the styles and apply them to the div instead. That should work for the one spot you need this style prop rn

Reply:
you don't need to give it a styles prop (in fact you definitely shouldn't since we want this component styled a particular way), so you should remove that. You can just wrap it in a div wherever you need to apply the styles and apply them to the div instead. That should work for the one spot you need this style prop rn

Reply:
you should however, add smthn to the button info prop that lets you put a divider above a button like there currently is between review and delete on the CR actions dropdown